src/main/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImpl.java

```diff
@@ -347,7 +325,7 @@ protected String validateACHBatchDetail(PayeeACHAccountExtractDetail achDetail,
             LOG.warn("validateACHBatchDetail:" + failureMessage.toString());
             return failureMessage.toString();
         }
-        return KFSConstants.EMPTY_STRING;
+        return StringUtils.EMPTY;

```

Comment:
I was trying to update this for consistency with much of our other recent code, where we are using StringUtils.EMPTY instead of the KFS-delivered constant. Let me know if you think it needs to be changed back.

Reply:
Deferring to Daniela as this would/could affect Kuali Co patches.

Reply:
I don't think it matters which constant we are using in this case. Base code is using both StringUtils.EMPTY and KFSConstants.EMPTY_STRING although it seems to favor the latter.